docs/cli-new-transform-compiled-releases.rst

```diff
@@ -0,0 +1,16 @@
+Command line tool - new-transform-compiled-releases option
```

Comment:
1) As commented elsewhere, the method of running both steps is not as good as is it could be. This is something I want to improve in the future, and thus having a separation makes it easier to improve each one independently.

2) Bearing in mind that transforms may take a long time to run, we need to make something that is resumable. Hence the has_ocid_been_transformed() function, and hence this separation - currently in this code, the second command can be run repeatedly with no bad effects.

Reply:
(1) sounds fine – I guess one possible outcome of the improvements might be to merge the commands. Regarding (2), the first command presently succeeds with "Already exists!" if the command had already been run. If the two commands were merged, I figure it would instead resume if the collection already existed. Anyhow, this can be determined later.